Formula/mysql.rb

```diff
@@ -123,13 +123,27 @@ def post_install
       system bin/"mysqld", "--initialize-insecure", "--user=#{ENV["USER"]}",
         "--basedir=#{prefix}", "--datadir=#{datadir}", "--tmpdir=/tmp"
     end
+
+    # Create my.cnf that binds to 127.0.0.1 by default
+    unless (etc/"my.cnf").exist?
+      s = <<-EOS.undent
+      # Default Homebrew MySQL server config
+      [mysqld]
+      # Only allow connections from localhost by default
+      bind-address = 127.0.0.1
+      EOS
+      File.write(etc/"my.cnf", s)
+    end
   end
 
   def caveats
     s = <<-EOS.undent
     We've installed your MySQL database without a root password. To secure it run:
         mysql_secure_installation
 
+    We've installed a default server config which only allows connections from localhost.
```

Comment:
`MySQL is configured to only allow connections from localhost by default` and skip mention of the configuration as it's in the default, typical location.